src/containers/offerContainer/offerContainer.tsx

```diff
@@ -0,0 +1,43 @@
+import Box from '@mui/material/Box'
+import OfferCard from '~/components/offer-card/OfferCard'
+import { CardsViewEnums } from '~/types/findOffers/enums/findOffers.enums'
+
+interface OfferCardsProps {
+  viewMode: CardsViewEnums
+  offerCards: {
```

Comment:
this should be in offer.interfases as OfferResponse

Reply:
Fixed.